src/select/styled-components.js

```diff
@@ -309,7 +309,6 @@ export const StyledInput = styled<SharedStylePropsArgT>('input', props => {
   return {
     ...getFont($size, typography),
     boxSizing: 'content-box',
-    fontSize: '16px', // prevents iOS to zoom in when focused
```

Comment:
should this be maintained so that ios does not zoom?

Reply:
oh, that's a really good catch, we should keep that

Reply:
The default input font size is 16px and for the large and compact ones it doesn't look good with the hardcoded 16px font.